packages/reports/tests/integration/components/filter-values/dimension-select-test.ts

```diff
@@ -363,8 +364,29 @@ module('Integration | Component | filter values/dimension select', function (hoo
         el.querySelector('.filter-values--dimension-select__option-value')?.textContent?.trim(),
         el.querySelector('.filter-values--dimension-select__option-context')?.textContent?.trim(),
       ]),
-      [['65 and over', 'id: 10']],
+      [
+        ['"over"', ''],
+        ['65 and over', 'id: 10'],
+      ],
       '`dimension-select` fetches values for new dimension field'
     );
   });
+
+  test('testing manual filter', async function (this: TestContext, assert) {
+    this.filter = this.fragmentFactory.createFilter('dimension', 'bardOne', 'age', { field: 'id' }, 'in', []);
+    this.onUpdateFilter = (changeSet: Partial<FilterFragment>) => {
+      this.set('filter.values', changeSet.values);
```

Comment:
Can we click the `'abc'` value and assert it is passed in as expected

Reply:
@kevinhinterlong What **click** helper would I use here to test?